practice4/src/test/java/com/academiaDigital/softwareArchitecture/product/CityTest.java

```diff
@@ -0,0 +1,28 @@
+package com.academiaDigital.softwareArchitecture.product;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CityTest {
+
+    @Test
+    public void getOrcHouses_should_return_array() {

```

Comment:
good that we have unit tests, Ideally we'd have a unit test for each class.

Reply:
Yes but in that case I have void methods, but I have changed some methods in order to have testeable methods